src/test/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRRepositoryTest.java

```diff
@@ -97,10 +96,8 @@ public void doClearRepoPullsDeleted() throws IOException {
         PowerMockito.mockStatic(BulkChange.class);
         when(BulkChange.contains(repo)).thenReturn(true);
 
-        FormValidation formValidation = repo.doClearRepo();
-
-        Assert.assertEquals(FormValidation.Kind.OK, formValidation.kind);
-        Assert.assertEquals(null, repo.getPulls());
+        assertThat(repo.doClearRepo().kind, equalTo(FormValidation.Kind.OK));
+        assertThat(repo.getPulls().keySet(), hasSize(0));
```

Comment:
Could you please explain why assertThat() in this two cases is better than previous variant with assertEquals()?